crates/tinymist-query/src/upstream/complete.rs

```diff
@@ -1119,9 +1130,10 @@ impl<'a, 'w> CompletionContext<'a, 'w> {
                         if close { ">" } else { "" }
                     )
                 }),
-                label_detail: label_desc,
                 label: label.as_str().into(),
+                label_detail: label_desc,
                 detail,
+                filter_text: Some(filter_text),
```

Comment:
I have tried another strategy, that separates another completion item for each bib title, and think of it would be better, since vscode will hightlight pattern matched in title for you:

![image](https://github.com/Myriad-Dreamin/tinymist/assets/35292584/c2ab25dc-efe4-4e50-b69b-db6f2c27310b)

![image](https://github.com/Myriad-Dreamin/tinymist/assets/35292584/edc0a592-f878-4975-a4b5-1a5a363cae52)

```rs
let label: EcoString = label.as_str().into();
let completion = Completion {
    kind: CompletionKind::Constant,
    apply: (open || close).then(|| {
        eco_format!(
            "{}{}{}",
            if open { "<" } else { "" },
            label.as_str(),
            if close { ">" } else { "" }
        )
    }),
    label: label.clone(),
    label_detail: label_desc,
    filter_text: Some(label.clone()),
    detail,
    ..Completion::default()
};

if let Some(bib_title) = bib_title {
    self.completions.push(Completion {
        label: bib_title.clone(),
        label_detail: Some(label),
        filter_text: Some(bib_title),
        ..completion.clone()
    });
};
self.completions.push(completion);
```